single-node/user-data

```diff
@@ -856,15 +951,14 @@ function start_addons {
     curl --silent -H "Content-Type: application/yaml" -XPOST -d"$(cat /srv/kubernetes/manifests/kube-dashboard-svc.yaml)" "http://127.0.0.1:8080/api/v1/namespaces/kube-system/services" > /dev/null
 }
 
-function enable_calico_policy {
+function enable_calico {
     echo "Waiting for Kubernetes API..."
     until curl --silent "http://127.0.0.1:8080/version"
     do
         sleep 5
     done
     echo
-    echo "K8S: Calico Policy"
-    curl --silent -H "Content-Type: application/json" -XPOST -d"$(cat /srv/kubernetes/manifests/calico-system.json)" "http://127.0.0.1:8080/api/v1/namespaces/" > /dev/null
+    docker run --rm --net=host -v /srv/kubernetes/manifests:/host/manifests $HYPERKUBE_IMAGE_REPO:$K8S_VER /hyperkube kubectl apply -f /host/manifests/calico.yaml
```

Comment:
I'm getting `device or resource busy` when running rm against that pod, the full set of commands:

```sh
/usr/bin/rkt run --uuid-file-save=/var/run/calico-install.uuid \
                        --net=host \
                        --volume manifests,kind=host,source=/etc/kubernetes/manifests \
                        --mount volume=manifests,target=/host/manifests \
                        $HYPERKUBE_IMAGE_REPO:$K8S_VER --exec=/hyperkube -- kubectl apply -f /host/manifests/calico.yaml
/usr/bin/rkt rm --uuid-file=/var/run/calico-install.uuid
```

Any suggestions?  Should I just leave the pod in an exited state?